bin/start-pgbouncer

```diff
@@ -151,8 +151,17 @@ app-start() {
   at "$name-launched pid=$!"
 }
 
+
+function inf() {
+  echo -e "INFO:  $* "
+}
+
+function err() {
+   echo -e "ERROR: $* "
+}
```

Comment:
Might make sense to abstract this into a single reusable function like `logToConsole('info', 'my_message')` so we don't have to keep defining it and so all uses cases remain stylistically the same.

Reply:
Yes normally I'd have a shared library but we wanted to keep this simple.